applications/serial_lte_modem/src/slm_at_tcp_proxy.c

```diff
@@ -567,14 +599,8 @@ static void tcpsvr_thread_func(void *p1, void *p2, void *p3)
 	ring_buf_reset(&data_buf);
 	while (true) {
 		if (k_timer_status_get(&conn_timer) > 0) {
-			k_timer_stop(&conn_timer);
-			LOG_INF("Connecion timeout");
-			sprintf(rsp_buf, "#XTCPSVR: %d,\"timeout\"\r\n",
-				-ETIME);
-			rsp_send(rsp_buf, strlen(rsp_buf));
-			close(proxy.sock_peer);
-			proxy.sock_peer = INVALID_SOCKET;
-			nfds--;
+			LOG_WRN("Connecion timeout");
+			tcp_terminate_connection(-ETIME);
```

Comment:
Could use more specific `ETIMEDOUT` here.

Reply:
ETIMEOUT not defined in newlib/errno.h
Agreed with Honeywell to notify with below error code before.
#define	ETIME		62	/* Timer expired */